common/djangoapps/adaptive_learning/admin.py

```diff
@@ -0,0 +1,8 @@
+"""
+Admin for Adaptive Learning
+"""
+from django.contrib import admin
+from adaptive_learning.config.models import AdaptiveLearningEnabledFlag
+
+admin.site.register(AdaptiveLearningEnabledFlag)
```

Comment:
I think we need `ConfigurationModelAdmin` as a 2nd argument here.

(`from config_models.admin import ConfigurationModelAdmin`).